src/osd/ECTransaction.h

```diff
@@ -120,6 +132,28 @@ class ECTransaction : public PGBackend::PGTransaction {
     assert(len == bl.length());
     ops.push_back(AppendOp(hoid, off, bl, fadvise_flags));
   }
+  void write(
```

Comment:
Yes, I will clean it up.

2016-02-03 21:53 GMT+08:00 Igor Fedotov <notifications@github.com>:

> In src/osd/ECTransaction.h
> <https://github.com/ceph/ceph/pull/7481#discussion_r51722407>:
>
> > @@ -120,6 +132,28 @@ class ECTransaction : public PGBackend::PGTransaction {
> >      assert(len == bl.length());
> >      ops.push_back(AppendOp(hoid, off, bl, fadvise_flags));
> >    }
> > +  void write(
>
> As you've started using write in EC backend I suggest to update write
> method definition at PGTransaction - make it pure virtual and remove out of
> comment saying "/// Optional, not supported on ec-pool" ( see PGBackend.h )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/ceph/pull/7481/files#r51722407>.
>